SECURITY.md

```diff
@@ -106,6 +106,10 @@ a security vulnerability. Examples of unwanted actions are polluting globals,
 causing an unrecoverable crash, or any other unexpected side effects that can
 lead to a loss of confidentiality, integrity, or availability.
 
+For example, if trusted input (like secure application code) is correct,
+then untrusted input must not lead to arbitrary JavaScript code execution or
+escape the sandbox.
```

Comment:
Is the term "sandbox" appropriate in this security context?

https://github.com/nodejs/node/blob/main/doc/contributing/security-model-strategy.md#document-the-security-model stated that Node.js is not a sandbox.

Reply:
Sandbox might not be the correct word to use here. Would changing it to:  must not lead to arbitrary JavaScript code execution or cross JavaScript expectations - be more clear?

Reply:
is "cross JavaScript expectations" a well-known term?

Reply:
Hm, I'm not sure, in my social circle, yes. Do you have any suggestion?

Reply:
No, I don't have good alternatives. Just an option: "must not lead to arbitrary code execution" (which should include side effects outside the process).

But I'm fine with yours at https://github.com/nodejs/node/pull/57426#discussion_r1993356509 as well.